deployment/app-chart/templates/astro-ui-configmap.yaml

```diff
@@ -6,6 +6,7 @@ data:
   APP_DOMAIN: "{{ .Values.environment.app_domain }}"
   DRAFTS_API_URL: "{{ .Values.environment.drafts_api_url }}"
   SEARCH_API_URL: "{{ .Values.environment.search_api_url }}"
+  API_GATEWAY_API_URL: "{{ .Values.environment.api_gateway_api_url }}"
```

Comment:
I am not sure where this `.Values.environment.api_gateway_api_url` is from. Is there a github setting for this?

Reply:
What is the purpose of this url? I cannot find amy place it used.

Reply:
That's for the BountyService to call the lambda
https://github.com/near-daos/astro-ui/blob/b4aa01920073eacfd825c1d03d0453b9758b2cf8/services/BountyService/BountyApiService.ts#L10-L16

Reply:
But we switched all opensearch related calls to api gateway already, I suppose you should refer to SEARCH_API_URL variable here as well.

Reply:
The name `SEARCH_API_URL` is confusing, as my BountyService has nothing to do with search. But I can reuse it for now. It will be great if you can rename it later.

Reply:
Those API_GATEWAY_API_URL have been removed. Please review again.